contracts/upgradeable_contracts/stake_token_mediators/BasicStakeTokenMediator.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.4.24;
+
+import "../amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol";
```

Comment:
Logically, `BasicAMBErc677ToErc677` is needed for ERC-TO-ERC bridges only. Also, STAKE bridge is a subtype of ERC-TO-ERC bridge. 
Then, I guess, it will be better to move `stake_bridge_mediators` into `amb_erc677_to_erc677` , so it will be `stake_bridge_mediators /amb_erc677_to_erc677`.

What do you think?

Reply:
I don't think that we will need the separate directory `stake_bridge_mediators` in this case. Consider to put everything in `amb_erc677_to_erc677` without subdirectories.